extension/src/accessors.rs

```diff
@@ -89,7 +89,14 @@ accessor! { first_val() }
 accessor! { last_val() }
 accessor! { first_time() }
 accessor! { last_time() }
-
+accessor! { open()}
+accessor! { close()}
+accessor! { high()}
+accessor! { low()}
+accessor! { open_time()}
+accessor! { high_time()}
+accessor! { low_time()}
+accessor! { close_time()}
```

Comment:
These all have a missing a space before the closing curly brace

Reply:
Fixed